semantic/src/definitions/modules/accordion.less

```diff
@@ -51,14 +51,14 @@
 }
 
 /* Default Styling */
-.ui.accordion:not(.styled) .title ~ .content:not(.ui),
-.ui.accordion:not(.styled) .accordion .title ~ .content:not(.ui) {
-  margin: @contentMargin;
-  padding: @contentPadding;
-}
-.ui.accordion:not(.styled) .title ~ .content:not(.ui):last-child {
-  padding-bottom: 0em;
-}
+// .ui.accordion:not(.styled) .title ~ .content:not(.ui),
+// .ui.accordion:not(.styled) .accordion .title ~ .content:not(.ui) {
+//   margin: @contentMargin;
+//   padding: @contentPadding;
+// }
+// .ui.accordion:not(.styled) .title ~ .content:not(.ui):last-child {
+//   padding-bottom: 0em;
+// }

```

Comment:
Note to reviewer: the reason this is commented out is quite inexplicable to me. Basically, when this is left as it was, the padding/margin values for the Accordion take on these values, _but only on Summary page!_ I've inspected the HTML and I can't see any difference in the component/class structure between Application and Summary pages that would affect this, but somehow it's taking on these ".not" properties on the Summary page. Any insights would be welcome.